uw-frame-components/portal/notifications/directives.js

```diff
@@ -11,16 +11,17 @@ define(['angular', 'require'], function(angular, require){
         }
     });
 
-    app.directive('notificationBell', function(){
+    app.directive('notificationPartials', function(){
```

Comment:
At the further risk of being pedantic: renaming a directive among the `uw-frame-components`, is that an API-breaking change? Necessitating a major version bump to uw-frame 3.0? (And hey, if we're bumping to 3.0 anyway, and we like the `ap-` namespace idea, opportunity to adopt that namespacing for all our directives. :smile:.